plugins/Jobs.cpp

```diff
@@ -1290,3 +1319,41 @@ bool BedrockPlugin_Jobs::_isValidSQLiteDateModifier(const string& modifier) {
     // Matched all parts, valid syntax
     return true;
 }
+
+void BedrockPlugin_Jobs::handleFailedReply(const BedrockCommand& command) {
+    if (SIEquals(command.request.methodLine, "GetJob") || SIEquals(command.request.methodLine, "GetJobs")) {
+
+        list<string> jobIDs;
+        if (SIEquals(command.request.methodLine, "GetJob")) {
+            STable jobJSON = SParseJSONObject(command.response.content);
+            if (jobJSON.find("jobID") != jobJSON.end()) {
+                jobIDs.push_back(jobJSON["jobID"]);
+            }
+        } else {
+            STable jobsJSON = SParseJSONObject(command.response.content);
+            list<string> jobs = SParseJSONArray(jobsJSON["jobs"]);
+            for (auto& job : jobs) {
+                STable jobJSON = SParseJSONObject(job);
+                if (jobJSON.find("jobID") != jobJSON.end()) {
+                    jobIDs.push_back(jobJSON["jobID"]);
+                }
+            }
+        }
+        SWARN("Failed sending response to '" << command.request.methodLine << "', re-queueing jobs: "<< SComposeList(jobIDs));
```

Comment:
Do we want to warn here? Given we know is "normal" and there's nothing else we can do, we should log hmmm or info.